### PR TITLE
Added a new Appwrite swag opportunity#1069

### PR DESCRIPTION
**Pull request by shivam200446**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I've updated the data.json file by adding the swag opportunity from Appwrite.

### CODE REVIEW

**Comment by shivam200446** [comment]
@swapagarwal & @aslafy-z  Sir Please look at issue and PR once
 thankyou!

File: data.json [hunk 1]
```diff
@@ -17,6 +17,15 @@
...
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Appwrite",
+    "difficulty": "Medium",
+    "description": "Earn an Appwrite hoodie and stickers for contributing only one accepted and merged PR!",
+    "reference": "https://appwrite.io/blog/post/hacktoberfest-2023",
+    "image": "https://appwrite.io/images/blog/hacktoberfest2023-swag.png",
+    "dateAdded": "2023-10-05",
+    "tags": ["clothing",  "Hoodie", "stickers"]
```
Comment (aslafy-z) [inline data.json:27]:
```suggestion
    "tags": ["clothing", "stickers"]
```

**Review by aslafy-z — APPROVED** [review]

**Comment by aslafy-z** [comment]
Nice swag, thank you @shivam200446

**Comment by shivam200446** [comment]
@aslafy-z Please add hacktoberfest-accepted label🙂

**Comment by shivam200446** [comment]
@aslafy-z Sir Please add hacktoberfest-accepted label🙂

**Comment by aslafy-z** [comment]
The hacktoberfest label is added at the repository level, there is no need for it to be added to the pull request.

Please check your [hacktoberfest profile](https://hacktoberfest.com/profile/), the PR should be marked as WAITING, which means the contribution was taken in account.

**Comment by shivam200446** [comment]
> The hacktoberfest label is added at the repository level, there is no need for it to be added to the pull request.
> 
> Please check your [hacktoberfest profile](https://hacktoberfest.com/profile/), the PR should be marked as WAITING, which means the contribution was taken in account.

Okey got it .. Thank you!

**Pull request merged**